VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+
+Even though delegator’s funds cannot be stolen by their validators, they are still liable if their validators misbehave. In such case, each delegator’s stake will be partially slashed in proportion of their relative stake.
+
+### What is the probability of being chosen as a proposer for any given block? Does it go up with the quantity of Atoms staked ?
+
+The validator that is selected to propose the next block is called proposer. Each proposer is selected deterministically, and the frequency of being chosen is equal to the relative total stake (where total stake = self-bonded stake + delegators stake) of the validator. For example, if the total bonded stake across all validators is 100 Atoms and a validator's total stake is 10 Atoms, then this validator has a 10% probability to be chosen as the next proposer.
+
+### What is the incentive to stake?
+
+Each member of a validator’s staking pool earns different types of rewards:
+* **Block provisions:** Native tokens of applications run by validators (e.g. Atoms) are inflated to produce block provisions. These provisions exist to incentivize Atom holders to bond their stake, as non-bonded Atom will be diluted over time. 
+* **Transaction fees:** The Hub maintains a whitelist of token that are accepted as fee payment. 
+
+These rewards are divided among validator’s staking pools according to each validator’s weight. Then, they are divided among delegators in proportion to each delegator’s stake. Note that a commission on fee rewards for delegators is applied by the validator before they are distributed.
+
+### What is the incentive to run a validator ?
+
+Validators earn proportionally more rewards than their delegators because of commissions on the fees that are collected. These fees can be collected not only in Atoms but in any token that has been whitelisted by Atom holders. 
+
+Validators also play a major role in governance. If a delegator does not vote, it inherits the vote from its validator. This gives validators a major responsibility in the ecosystem.
+
+### What is a validator's commission?
+
+Fee rewards received by a validator’s pool are split between the validator and its delegators. The validator can apply a commission on the part of the fees that goes to its delegators. This commission is set as a percentage. Each validator is free to set its initial commission, maximum daily commission change rate and maximum commission. The Cosmos Hub enforces the parameter set that each validator self defines. These parameters can only be defined when initially declaring candidacy. 
+
+Note that validators cannot set commission on block provisions. This is to prevent a continuous shift in distribution from delegators to validators which would further the concentration of ATOMs and ultimately weaken the security of the system.
```

Comment:
`... would increase the concentration of Atoms and ultimately weaken the security of the Cosmos Hub.`

Reply:
ok